website/docs/docs/mesh/iceberg/snowflake-iceberg-support.md

```diff
@@ -8,7 +8,15 @@ description: Understand Snowflake support for Apache Iceberg.
 dbt supports materializing the table in Iceberg table format in two different ways:
 
 - The model configuration field `table_format = 'iceberg'` (legacy)
-- Catalog integration in the model, resource, or dbt_project.yml configuration
+- Catalog integration can be configured in the SQL config (inside the `.sql` model file), property file (model folder), or project file ([`dbt_project.yml`](/reference/dbt_project.yml)).
+
+:::tip **Catalog integration configuration**
+
+You can also create a `catalogs.yml` at the [top level of your dbt project](/docs/mesh/iceberg/snowflake-iceberg-support#configure-catalog-integration-for-managed-iceberg-tables) by applying the catalog configuration at either the property file, folder, or your `dbt_project.yml`.
```

Comment:
In addition, I think this  "property file, folder, or your `dbt_project.yml`."

Should match what we say above or an abridged version of above? Something like:
> by applying the catalog configuration to the SQL config, property file, or your project file.